randovania/games/prime/patcher_file_lib/hint_name_creator.py

```diff
@@ -31,12 +32,15 @@ def node_name(self, pickup_node: PickupNode, hide_area: bool) -> str:
             )
 
 
-def color_text_as_red(text: str) -> str:
-    return "&push;&main-color=#a84343;{}&pop;".format(text)
+class TextColor(Enum):
+    GUARDIAN = "#FF3333"
+    ITEM = "#FF6705B3"
+    JOKE = "#45F731"
+    LOCATION = "#FF3333"
 
 
-def color_as_joke(text: str) -> str:
-    return "&push;&main-color=#45f731;{}&pop;".format(text)
+def color_text(color: TextColor, text:str):
```

Comment:
This could be a `TextColor` method.

Reply:
I know I considered that when I was writing this. I don't remember the exact reason I didn't do that; if I were to guess, I'd say that it probably felt out of scope for what the enum should contain.